src/EnergyPlus/api/autosizing.py

```diff
@@ -0,0 +1,100 @@
+        self.api.sizerHeatingAirflowUAInitializeForSystemHeatingDuct.restype = c_void_p
+        self.api.sizerHeatingAirflowUAInitializeForSystemOtherDuct.argtypes = [c_void_p, RealEP]
+        self.api.sizerHeatingAirflowUAInitializeForSystemOtherDuct.restype = c_void_p
+        self.api.sizerHeatingAirflowUADelete.argtypes = [c_void_p]
+        self.api.sizerHeatingAirflowUADelete.restype = c_void_p
+        self.api.sizerHeatingAirflowUACalculate.argtypes = [c_void_p]
+        self.api.sizerHeatingAirflowUACalculate.restype = int
+        self.api.sizerHeatingAirflowUAValue.argtypes = [c_void_p]
+        self.api.sizerHeatingAirflowUAValue.restype = RealEP
+        self.instance = self.api.sizerHeatingAirflowUANew()
+
+    def __del__(self):
+        self.api.sizerHeatingAirflowUADelete(self.instance)
+
+    def initialize(self, elevation: float = 0.0) -> None:
+        """
+        Performs initialization of the sizer, eventually it will have lots of args
+
+        :param elevation: Elevation above sea level for evaluating fluid properties, in meters, default zero
+        :return: Nothing
+        """
+        self.api.sizerHeatingAirflowUAInitialize(self.instance, elevation)
+
+    def initialize_for_zone_terminal_single_duct(self, elevation: float, main_flow_rate: float) -> None:
+        self.api.sizerHeatingAirflowUAInitializeForSingleDuctZoneTerminal(self.instance, elevation, main_flow_rate)
+
+    def initialize_for_zone_terminal_induction_unit(self, elevation: float, main_flow_rate: float, reheat_mult: float) -> None:
+        self.api.sizerHeatingAirflowUAInitializeForZoneInductionUnit(self.instance, elevation, main_flow_rate, reheat_mult)
+
+    def initialize_for_zone_terminal_fan_coil(self, elevation: float, design_heat_volume_flow_rate: float) -> None:
+        self.api.sizerHeatingAirflowUAInitializeForZoneFanCoil(self.instance, elevation, design_heat_volume_flow_rate)
+
+    def initialize_for_system_outdoor_air(self, elevation: float, overall_system_mass_flow: float, doas: bool) -> None:
+        self.api.sizerHeatingAirflowUAInitializeForSystemOutdoorAir(self.instance, elevation, overall_system_mass_flow, 1 if doas else 0)
+
+    def initialize_for_system_main_duct(self, elevation: float, overall_system_vol_flow: float, min_flow_rate_ratio: float) -> None:
+        self.api.sizerHeatingAirflowUAInitializeForSystemMainDuct(self.instance, elevation, overall_system_vol_flow, min_flow_rate_ratio)
+
+    def initialize_for_system_cooling_duct(self, elevation: float) -> None:
+        self.api.sizerHeatingAirflowUAInitializeForSystemCoolingDuct(self.instance, elevation)
+
+    def initialize_for_system_heating_duct(self, elevation: float) -> None:
+        self.api.sizerHeatingAirflowUAInitializeForSystemHeatingDuct(self.instance, elevation)
+
+    def initialize_for_system_other(self, elevation: float) -> None:
+        self.api.sizerHeatingAirflowUAInitializeForSystemOtherDuct(self.instance, elevation)
```

Comment:
Once we settle on the initialization stuff, I'll document the Python functions here.